pulp_smash/tests/rpm/api_v2/broker_reconnect.py

```diff
@@ -0,0 +1,147 @@
+# coding=utf-8
+"""Tests for Pulp's "broker reconnect" feature.
+
+Tests for `#55 <https://github.com/PulpQE/pulp-smash/issues/55>`_:
+
+> Pulp offers a collection of behaviors known as "reconnect support" for the
+> Pulp Broker. Here are the expected behaviors:
+>
+> * If you start a Pulp service that connects to the broker and the broker is
+>   not running or is not network accessible for some reason, the Pulp services
+>   will wait-and-retry. It has a backoff behavior, but the important part is
+>   that it doesn't exit if it can't connect due to availability and when the
```

Comment:
I think I wrote this originally...
s/that it doesn't/that Pulp services don't/
s/if it/if they/
s/ and when/, and when/